portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/TextInput.path

```diff
@@ -271,8 +271,8 @@
 	<td></td>
 </tr>
 <tr>
-	<td>ENTER_AGAIN</td>
-	<td>//div[label[contains(.,'Enter Again')]]/input</td>
+	<td>REENTER_PASSWORD</td>
+	<td>//div[label[contains(.,'Reenter Password')]]/input</td>
```

Comment:
Since the text label changed to "Reenter Password", I just updated the path and all the macros and tests that use it.